pkg/config/otelconfig/otelconfig.go

```diff
@@ -106,7 +104,10 @@ func create(jaeger *v1.Jaeger, component string, otelConfig map[string]interface
 
 // Update injects required flags and objects to the common spec.
 func Update(jaeger *v1.Jaeger, component string, commonSpec *v1.JaegerCommonSpec, args *[]string) {
-	volume := corev1.Volume{
+	configFileLocation := fmt.Sprintf("/etc/jaeger/otel/%s/", component)
```

Comment:
nit: is there a to fix this without embedding the component name in the path? I find it redundant but I might be missing something.

Reply:
I reverted this change, at first I thought it was part of the problem but it actually wasn't.